components/index.ts

```diff
@@ -5,4 +5,5 @@ export { default as VideoPlayerStream } from './video-player-stream'
 export { Checkbox, CheckIndicator } from './checkbox'
 export { Avatar, AvatarImage, AvatarFallback } from './avatar'
 export { RadioButtons, RadioButton, RadioIndicator } from './radio-buttons'
+export { Modal as ModalRadix } from './modal-radix'
```

Comment:
We can just call this "Modal". Currently, the plan is to use Radix for all the components in our library so there's no need to call it out specifically. All the components are Radix under the hood. Same with the file name - it can just be modal.tsx

Reply:
Named it this way because I saw another Modal in the components folder that is being used on the launch page.  Hoping to find out from @replygirl how I can simulate the live stream so I can replace that modal with the Radix one.  Once that's done I'd name as modal and remove the other one.

Reply:
@jackhajb you should be replacing the guts of my modal component to whatever extent is necessary to get the accessibility improvements radix is offering here, and compensate for any impacts that has to the stream player--so `modal-radix.tsx` goes away and most or all of its contents go to `modal.tsx`. on preview deployments, the live stream is enabled, so if you go to `/`, you'll see the option to join the stream

Reply:
@replygirl Removed the old modal and replaced with the radix version.